pandas/core/categorical.py

```diff
@@ -1862,8 +1990,11 @@ def convert_categorical(x):
     if get_dtype_kinds(to_concat) - set(['object', 'category']):
         # convert to object type and perform a regular concat
         from pandas.core.common import _concat_compat
-        return _concat_compat([np.array(x, copy=False, dtype=object)
-                               for x in to_concat], axis=0)
```

Comment:
I found a config that shows promise.  Basically setting `split_penalty_after_opening_bracket` to a very high number (`unbreakable` in yapf) fixes many of these open bracket problems.  Nevertheless, the long error strings still need to be manually split (as they did before).  Long comments still need to be split as well.  There is manual work but I think it's a large one-time investment to get 98% of the way to cleaner code that now has no pep8 warnings.  In this PR I did edit many of them manually but there are only a few left that need changing and hopefully this configuration limits that even further.  I'll fix these and then hopefully subsequent ones are less manual and painful.

Base config:
```
            warn(
                "Creating a 'Categorical' with 'levels' is deprecated, use 'categories' instead",
                FutureWarning,
                stacklevel=2)
...
        return _concat_compat(
            [np.array(x,
                      copy=False,
                      dtype=object) for x in to_concat],
            axis=0)

```

New config (who needs to be split):
```
            warn("Creating a 'Categorical' with 'levels' is deprecated, use 'categories' instead",
                 FutureWarning,
                 stacklevel=2)
...
        return _concat_compat([np.array(x, copy=False, dtype=object)
                               for x in to_concat], axis=0)
```